src/functions-reference/array_operations.Rmd

```diff
@@ -265,73 +265,73 @@ an array of a specified size.  These operations work for all input
 types T, including reals, integers, vectors, row vectors, matrices, or
 arrays.
 
-<!-- T[]; rep_array; (T x, int n); -->
-\index{{\tt \bfseries rep\_array }!{\tt (T x, int n): T[]}|hyperpage}
+<!-- array[] T; rep_array; (T x, int n); -->
```

Comment:
These commented forms are for export to things like RStudio.  Are you sure they can handle this `array[] T` format?  If not, @bgoodri or @jgabry may know what RStudio needs.

Reply:
We'll have to add signatures for that.

Reply:
They were simply caught by the find-and-replace done for the formatted output. I was assuming the comments were there as a more human-readable note in the source text, that's my bad. I can change them back if it's too much of a problem?